docs/guides/remove-trusted-relayer.mdx

```diff
@@ -1,4 +1,4 @@
-# How to Remove the Trusted Relayer ISM from your Hyperlane Warp Route
+# How to Remove Warp Route Trusted Relayer ISM
```

Comment:
A bit too long to fit on the page

Reply:
imo the original wording is necessary for consistency in the Get Started section– and I think the title wraps nicely when it's long, no?

Reply:
Imo, we dont need to mention Hyperlane in the title. The reader should know its Hyperlane. But happy to change it back for consistency